src/cltk/morphology/universal_dependencies_features.py

```diff
@@ -399,7 +399,7 @@ class NameType(MorphosyntacticFeature):
     other = auto()
 
 
-class PrononimalType(MorphosyntacticFeature):
+class PronominalType(MorphosyntacticFeature):
```

Comment:
This change may break compatibility with previous versions. I'm sure that this is barely used but this change can lead to unexepected errors. I'm not sure that changes may be a good thing. Maybe for the future.

Reply:
Ok, this typo must be fixed. I accept this change.